### PR TITLE
Update configure-advanced-delivery.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the module used as some of them need Security & Compliance PowerShell

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9606

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 1]
```diff
@@ -329,7 +329,7 @@ Configuring a third-party phishing simulation in PowerShell is a multi-step proc
 
 #### Step 1: Use PowerShell to create the phishing simulation override policy
 
-In [Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell), this example creates the phishing simulation override policy.
+In [Security & Compliance PowerShell](https://docs.microsoft.com/powershell/exchange/scc-powershell), this example creates the phishing simulation override policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
In [Security & Compliance PowerShell](/powershell/exchange/scc-powershell), this example creates the phishing simulation override policy.
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 2]
```diff
@@ -341,7 +341,7 @@ For detailed syntax and parameter information, see [New-PhishSimOverridePolicy](
 
 #### Step 2: Use PowerShell to create the phishing simulation override rule
 
-In [Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell), use the following syntax:
+In [Security & Compliance PowerShell](https://docs.microsoft.com/powershell/exchange/scc-powershell), use the following syntax:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
In [Security & Compliance PowerShell](/powershell/exchange/scc-powershell), use the following syntax:
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 3]
```diff
@@ -383,7 +383,7 @@ For detailed syntax and parameter information, see [New-TenantAllowBlockListItem
 
 ### Use PowerShell to view the phishing simulation override policy
 
-In [Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell), this example returns detailed information about the one and only phishing simulation override policy.
+In [Security & Compliance PowerShell](https://docs.microsoft.com/powershell/exchange/scc-powershell), this example returns detailed information about the one and only phishing simulation override policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
In [Security & Compliance PowerShell](/powershell/exchange/scc-powershell), this example returns detailed information about the one and only phishing simulation override policy.
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 4]
```diff
@@ -393,7 +393,7 @@ For detailed syntax and parameter information, see [Get-PhishSimOverridePolicy](
 
 ### Use PowerShell to view phishing simulation override rules
 
-In [Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell), this example returns detailed information about phishing simulation override rules.
+In [Security & Compliance PowerShell](https://docs.microsoft.com/powershell/exchange/scc-powershell), this example returns detailed information about phishing simulation override rules.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
In [Security & Compliance PowerShell](/powershell/exchange/scc-powershell), this example returns detailed information about phishing simulation override rules.
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 5]
```diff
@@ -423,7 +423,7 @@ For detailed syntax and parameter information, see [Get-TenantAllowBlockListItem
 
 ### Use PowerShell to modify the phishing simulation override policy
 
-In [Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell), use the following syntax:
+In [Security & Compliance PowerShell](https://docs.microsoft.com/powershell/exchange/scc-powershell), use the following syntax:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
In [Security & Compliance PowerShell](/powershell/exchange/scc-powershell), use the following syntax:
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 6]
```diff
@@ -439,7 +439,7 @@ For detailed syntax and parameter information, see [Set-PhishSimOverridePolicy](
 
 ### Use PowerShell to modify phishing simulation override rules
 
-In [Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell), use the following syntax:
+In [Security & Compliance PowerShell](https://docs.microsoft.com/powershell/exchange/scc-powershell), use the following syntax:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
In [Security & Compliance PowerShell](/powershell/exchange/scc-powershell), use the following syntax:
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 7]
```diff
@@ -480,7 +480,7 @@ For detailed syntax and parameter information, see [Set-TenantAllowBlockListItem
 
 ### Use PowerShell to remove a phishing simulation override policy
 
-In [Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell), this example removes the phishing simulation override policy and the corresponding rule.
+In [Security & Compliance PowerShell](https://docs.microsoft.com/powershell/exchange/scc-powershell), this example removes the phishing simulation override policy and the corresponding rule.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
In [Security & Compliance PowerShell](/powershell/exchange/scc-powershell), this example removes the phishing simulation override policy and the corresponding rule.
```

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 8]
```diff
@@ -490,7 +490,7 @@ For detailed syntax and parameter information, see [Remove-PhishSimOverridePolic
 
 ### Use PowerShell to remove phishing simulation override rules
 
-In [Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell), use the following syntax:
+In [Security & Compliance PowerShell](https://docs.microsoft.com/powershell/exchange/scc-powershell), use the following syntax:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md:493]:
```suggestion
In [Security & Compliance PowerShell](https://docs.microsoft.com/powershell/exchange/scc-powershell), use the following syntax:
```
```suggestion
In [Security & Compliance PowerShell](/powershell/exchange/scc-powershell), use the following syntax:
```

Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
In [Security & Compliance PowerShell](/powershell/exchange/scc-powershell), use the following syntax:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**